### PR TITLE
adds forbidden lore to emagged library consoles

### PR DESCRIPTION
**Pull request by Cenrus**
The emagged library console is now capable of making clockwork slabs, alongside blood daggers. Useless if you're not a cultist, but can create a cult scare among the station.

:cl:
add: Emagged library console can now make heretic books and clockwork slabs.
/:cl:

### CODE REVIEW

**Comment by VictorVow** [comment]
Why a slab? Makes more sense that it'd be able to print the heretic's codex if anything.

**Comment by ZombiesVsPlants** [comment]
> Why a slab? Makes more sense that it'd be able to print the heretic's codex if anything.

Because you can also print cultist books in the emagged library console.
Very nice touch I look forward to the trolling.

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -322,7 +322,10 @@ GLOBAL_LIST(cachedbooks) // List of our cached book datums
 	if (prob(50))
 		new /obj/item/melee/cultblade/dagger(get_turf(src))
 		to_chat(user, "<span class='warning'>Your sanity barely endures the seconds spent in the vault's browsing window. The only thing to remind you of this when you stop browsing is a sinister dagger sitting on the desk. You don't even remember where it came from...</span>")
-
+	else
+		new /obj/item/clockwork/clockwork_slab(get_turf(src))
+		to_chat(user, "<span class='warning'>Your sanity barely endures the seconds spent in the vault's browsing window. The only thing to remind you of this when you stop browsing is a strange metal tablet sitting on the desk. You don't even remember where it came from...</span>")
+
```
Comment (DatBoiTim) [inline code/modules/library/lib_machines.dm]: Why not also add heretic book as well with two prob(33)s and an else?

Comment (francinum) [reply code/modules/library/lib_machines.dm]: It'd be far better to have a switch statement at that point.

File: code/modules/library/lib_machines.dm [hunk 2]
```diff
@@ -319,10 +319,17 @@ GLOBAL_LIST(cachedbooks) // List of our cached book datums
 	return locate(/obj/machinery/libraryscanner) in range(viewrange, get_turf(src))
 
 /obj/machinery/computer/libraryconsole/bookmanagement/proc/print_forbidden_lore(mob/user)
-	if (prob(50))
-		new /obj/item/melee/cultblade/dagger(get_turf(src))
-		to_chat(user, "<span class='warning'>Your sanity barely endures the seconds spent in the vault's browsing window. The only thing to remind you of this when you stop browsing is a sinister dagger sitting on the desk. You don't even remember where it came from...</span>")
-
+	var/selection = rand(1,3)
+	switch(selection)
```
Comment (utilbird) [inline code/modules/library/lib_machines.dm]:
```suggestion
	switch(rand(1,3))
```

**Review by utilbird — APPROVED** [review]

**Pull request merged**